querydsl-jpa/src/test/java/com/querydsl/jpa/AbstractJPATest.java

```diff
@@ -318,6 +368,12 @@ public void Case2() {
                         .select(Expressions.cases().when(cat.toes.eq(2)).then(cat.id.multiply(2))
                                 .when(cat.toes.eq(3)).then(cat.id.multiply(3))
                                 .otherwise(4)).fetch());
+
+        List<Integer> rv = query().from(cat)
+            .select(Expressions.cases().when(cat.toes.eq(2)).then(cat.id.multiply(2))
```

Comment:
This query is already run by the [statement above](https://github.com/querydsl/querydsl/pull/1208/files#diff-7d971dd1c9376907a88a32f2d2c5881cR368) right?